src/Modules/Preferences.cs

```diff
@@ -0,0 +1,315 @@
+﻿using System;
+using System.Reflection;
+using KSP.IO;
+
+namespace KERBALISM
+{
+	public class PreferencesBasic : GameParameters.CustomParameterNode
+	{
+		[GameParameters.CustomParameterUI("Stock Science Dialog", toolTip = "Keep showing the stock science dialog")]
+		public bool scienceDialog = true;
+
+		[GameParameters.CustomParameterUI("Highlight Malfunctions", toolTip = "Highlight faild parts in flight")]
+		public bool highlights = true;
+
+		[GameParameters.CustomIntParameterUI("Breakdown Penalty", minValue = 0, maxValue = 300, toolTip = "Reputation removed when a Kerbal looses his marbles in space")]
+		public int breakdownPenalty = 10;
+
+		[GameParameters.CustomIntParameterUI("Death Penalty", minValue = 0, maxValue = 300, toolTip = "Reputation removed when a Kerbal dies")]
+		public int deathPenalty = 100;
+
+		[GameParameters.CustomParameterUI("Incentive Redundancy", toolTip = "Each malfunction will increase the MTBF\nof components in the same redundancy group")]
+		public bool incentiveRedundancy = true;
+
+		[GameParameters.CustomFloatParameterUI("Critical Failure Rate", asPercentage = true, minValue = 0, maxValue = 1, displayFormat = "F2", toolTip = "Proportion of malfunctions that lead to critical failures")]
+		public float criticalChance = 0.25f;
+
+		[GameParameters.CustomFloatParameterUI("Fixable Failure Rate", asPercentage = true, minValue = 0, maxValue = 1, displayFormat = "F2", toolTip = "Proportion of malfunctions that can be fixed remotely")]
+		public float safeModeChance = 0.5f;
+
+		public override GameParameters.GameMode GameMode { get { return GameParameters.GameMode.ANY; } }
+
+		public override bool HasPresets { get { return false; } }
+
+		public override string DisplaySection { get { return "Kerbalism"; } }
+
+		public override string Section { get { return "Kerbalism"; } }
+
+		public override int SectionOrder { get { return 0; } }
+
+		public override string Title { get { return "Common"; } }
+
+		private static PreferencesBasic instance;
+
+		public static PreferencesBasic Instance

```

Comment:
maybe rename this to fetch to fit in better with KSP coding style, no idea why squad thought it a good idea to do such a thing 🤷‍♂️

Reply:
same with the other occurrences of Instance

Reply:
Actually I stole that bit from kOS. No idea why they named it that way... Instance refers to the singleton pattern I use a lot in different languages.
I'm new to C#, so bear with me...

Reply:
Instance should also be used also in C#, its just that Squad uses fetch for some unknown reason